lib/common/zstd_internal.h

```diff
@@ -137,7 +137,10 @@ static const U32 OF_defaultNormLog = 5;
 /*-*******************************************
 *  Shared functions to include for inlining
 *********************************************/
-static void ZSTD_copy8(void* dst, const void* src) { memcpy(dst, src, 8); }
+static void ZSTD_copy8(void* dst, const void* src)
+{
+    MEM_write64(dst, MEM_read64(src));
```

Comment:
If you are confident about that I can drop this patch.  But I still think it is inconsistent to always use `memcpy()` here, despite it potentially being disabled in `mem.h`.